hutch_python/qs_load.py

```diff
@@ -16,6 +26,90 @@
 logger = logging.getLogger(__name__)
 
 
+# Annotation with dataclass, making struct to help organize cds objects in prettytable
+@dataclass
+class QStruct:
+    alias: str
+    pvbase: str
+    pvtype: str
+
+
+def pull_cds_items(exp, run):
+    """
+    Gather all user obejcts from the CDS tab in the questionnaire.
+    Parse objects and sperate them based on type.
+    Display them in the console vie PrettyTable.
+
+    Parameters
+    ----------
+    exp: ``str``
+        The experiment's name e.g. xppx1003221
+    run: ''str''
+        The run number e.g. run21
+    ----------
```

Comment:
DOC: I think these extra ------- lines will mess with the autodocs, but I'm not 100% sure
See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html